index.html

```diff
@@ -80,7 +80,8 @@ <h2>Featured Items</h2>
         <div class="item-cards-container">
 
         </div>
-        <div class="morebtn" onclick="moreCards()">MORE <i class="fa-solid fa-angle-down"></i></div>
+        <div class="morebtn more" onclick="moreCards()">MORE <i class="fa-solid fa-angle-down"></i></div>
+        <div class="morebtn less hide-item" onclick="moreCards()">LESS <i class="fa-solid fa-angle-up"></i></div>
     </section>
     <section id="partners" class="footer-partners">
```

Comment:
- [ ] **Required** In the `Partner` section there should be the logo image of the partner, Kindly remove the `daumkakao` text & add the logo there.
- Your design
![Screenshot from 2022-04-14 17-32-45](https://user-images.githubusercontent.com/26767383/163402345-00f6eb7e-20ec-40dd-8cad-005bd6af1044.png)

- Required design 
![Screenshot from 2022-04-14 17-33-02](https://user-images.githubusercontent.com/26767383/163402398-c96d78f4-650a-4d98-9ab3-cfc2e957a7c3.png)